plugins/modules/gitlab_runner.py

```diff
@@ -86,11 +86,10 @@
       - Determines if a runner can pick up jobs only from protected branches.
       - If I(access_level_on_creation) is not explicitly set to C(true), this option is ignored on registration and
         is only applied on updates.
-      - If set to C(ref_protected), runner can pick up jobs only from protected branches.
       - If set to C(not_protected), runner can pick up jobs from both protected and unprotected branches.
+      - If set to C(ref_protected), runner can pick up jobs only from protected branches.
```

Comment:
This should say something like:
```suggestion
      - If set to C(ref_protected), runner can pick up jobs only from protected branches.
      - The current default is C(ref_protected). This will change to no default in community.general 8.0.0.
        From then on, there will be no default. When this option is not specified, GitLab will use C(not_protected)
        on creation, and the value set for the runner will not be changed on updates.
```

Reply:
Thanks a lot. I have extended the documentation section accordingly.